src/Management/Resources/Item.php

```diff
@@ -3,13 +3,139 @@
 namespace Doofinder\Management\Resources;
 
 use Doofinder\Configuration;
+use Doofinder\Shared\Exceptions\ApiException;
 use Doofinder\Shared\Interfaces\HttpClientInterface;
+use Doofinder\Shared\Interfaces\HttpResponseInterface;
 use Doofinder\Shared\Resource;
+use Doofinder\Management\Model\Item as ItemModel;
 
+/**
+ * Item class is responsible for making the requests to the item's endpoints and return a response
+ */
 class Item extends Resource
 {
+    /**
+     * @param HttpClientInterface $httpClient
+     * @param Configuration $config
+     * @return Item
+     */
     public static function create(HttpClientInterface $httpClient, Configuration $config)
     {
         return new self($httpClient, $config);
     }
+
+    /**
+     * @param string $hashId
+     * @param string $indexName
+     * @param string|null $itemId
+     * @return string
+     */
+    private function getBaseUrl($hashId, $indexName, $itemId = null)
+    {
+        // TODO To put /api/v2??
```

Comment:
Good question!! I forgot to resolve this point.

This is a doubt I had, because I don't know what url we provide to the client.
- https://lo_que_sea/api/v2
- https://lo_que_sea
In the second case I should put '/api/v2' string concatenated in the base url. (I'm pretty sure this is the way, so I should to change it)

@sonic182 I am right?

Reply:
The customer has the redoc documentation, you should use /api/v2 as we will stick to that management api version